### DISCUSSION TITLE
Auto upgrade pet

### DISCUSSION THREAD

**Discussion by Maxhem2** [thejoabo/virtualfisher-bot#90] [Ideas]
Instead of setting a pet, let it use the best bet it the user has

**Comment by thejoabo** [comment]
Hello @Maxhem2 , I don't know about that. I think the best solution would be to add a "_auto_" parameter (that would be the default one) to select the best available to the user, but still allow users to select manually the pet, because different pets have different uses: 
![image](https://user-images.githubusercontent.com/49594896/219880917-1c656548-5f4a-4832-aa74-c034b28c20bf.png)

**Reply by Maxhem2** [reply]
Ok now I see, I think there are different level Pets so it would be good if it automatically chose the best level pet (the one with the biggest bonus) and lets say I want the Tuna Dolphin buffer but currently don't have one the bot could autos elect any other pet until it gets the selected pet(dolphine in this example)

**Reply by Maxhem2** [reply]
Not really needed tbh
